### PR TITLE
CRT-476 Fix overflow scrolling in Safari

### PR DESCRIPTION
**Pull request by jacobp100**
https://ag-grid.atlassian.net/browse/CRT-476

Caused by the element shown extending the bounds of the page. It did this because it was placed in a container smaller than the page width, and tried to match the page width via `100vw`. Refactors the layout to give this element the natural full width of the page

image

### CODE REVIEW

File: external/ag-website-shared/src/components/license-pricing/license-pricing.module.scss [hunk 1]
```diff
@@ -15,17 +15,26 @@
...
     flex-direction: column;
-    margin-bottom: $spacing-size-8;
+    padding-top: $spacing-size-12 + ($spacing-size-1 / 2);
+    background: linear-gradient(
+        to bottom,
+        transparent,
+        color-mix(in srgb, var(--background) 5%, transparent) ($spacing-size-12 + 64px / 2),
+        transparent 0
+    );
+    --background: var(--color-bg-brand-solid);
```
Comment (markdurrant) [inline external/ag-website-shared/src/components/license-pricing/license-pricing.module.scss]: Minor nitpick. Define CSS vars at the top of declaration block.

**Review by markdurrant — APPROVED** [review]
One minor nitpick. Otherwise LGTM 👍

File: external/ag-website-shared/src/components/license-pricing/license-pricing.module.scss [hunk 2]
```diff
@@ -15,17 +15,26 @@
...
     display: flex;
     align-items: center;
     flex-direction: column;
-    margin-bottom: $spacing-size-8;
+    padding-top: $spacing-size-12 + ($spacing-size-1 / 2);
+    background: linear-gradient(
+        to bottom,
+        transparent,
+        color-mix(in srgb, var(--background) 5%, transparent) ($spacing-size-12 + 64px / 2),
```
Comment (markdurrant) [inline external/ag-website-shared/src/components/license-pricing/license-pricing.module.scss]: "Naked" slash division has be deprecated by Sass. Will cause console warnings. 

See https://sass-lang.com/d/slash-div for info.

**Review by markdurrant — CHANGES_REQUESTED** [review]
"Naked" slash division has be deprecated by Sass. Will cause console warnings.

File: external/ag-website-shared/src/components/license-pricing/license-pricing.module.scss (hunk 2 above)
Comment (jacobp100) [reply external/ag-website-shared/src/components/license-pricing/license-pricing.module.scss]: Updated

File: external/ag-website-shared/src/components/license-pricing/license-pricing.module.scss (hunk 1 above)
Comment (jacobp100) [reply external/ag-website-shared/src/components/license-pricing/license-pricing.module.scss]: Updated

**Review by markdurrant — APPROVED** [review]
Updates look good 👍

**Review by alantreadway — APPROVED** [review]

**Pull request merged**